docs/products/kafka/kafka-connect/howto/manage-connector-versions.md

```diff
@@ -0,0 +1,306 @@
+  enabled
+- [Aiven CLI](/docs/tools/cli)
+- [Aiven API](/docs/tools/api)
+- [Aiven Provider for Terraform](/docs/tools/terraform)
+
+:::note
+Multi-version support is only available for **dedicated Kafka Connect services**. Ensure
+that your service is updated to enable this feature.
+:::
+
+## Check available connector versions {#check-available-connector-versions}
+
+Before pinning a connector version, check which versions are available for your
+Apache Kafka Connect service. Verify if a connector supports multi-versioning by
+reviewing the listed versions. Use one of the following methods:
+
+<Tabs groupId="check-method">
+
+<TabItem value="api" label="Aiven API" default>
+
+1. Run the following command:
+
+   ```bash
+   curl -X GET "https://api.aiven.io/v1/project/<project_name>/service/<service_name>/available-connectors" \
+   -H "Authorization: Bearer <API_TOKEN>"
+   ```
+
+1. Review the response to see the available versions. If multiple versions are listed,
+   the connector supports multi-versioning.
+
+   Example output:
+
+   ```json
+   {
+    "plugin_name": "aiven-kafka-connect-jdbc",
+    "available_versions": [
+        {
+            "version": "6.10.0",
+            "deprecated": false
+        },
+        {
+            "version": "6.9.0",
+            "deprecated": true
+        }
+    ]
+   }
+   ```
+
+</TabItem>
+<TabItem value="cli" label="Aiven CLI">
+
+1. Run the following command:
+
+   ```bash
+   avn service connector available <service_name>
+   ```
+
+1. Review the output to see the available versions. If multiple versions are listed,
+   the connector supports multi-versioning.
+
+   Example output:
+
+   ```json
+   {
+      "plugin_name": "aiven-kafka-connect-jdbc",
+      "available_versions": [
+        {
+          "version": "6.10.0",
+          "deprecated": false
+        },
+        {
+          "version": "6.9.0",
+          "deprecated": true
+        }
+      ]
+    }
+   ```
+
+</TabItem>
+<TabItem value="terraform" label="Terraform">
+
+In Terraform, you can specify the desired connector version using the `plugin_versions`
+attribute in your configuration. This ensures that your Apache Kafka Connect service
+uses the exact version you need.
+
+For detailed steps and examples, see [Pin a connector version](#pin-version).
+
+</TabItem>
+</Tabs>
+
+## Pin a connector version {#pin-version}
+
+To use a specific connector version, update the `plugin_versions` property in the
+service configuration. Pinning a **plugin version** applies to all connectors
+(source and sink) provided by that plugin. For example, pinning
+the `aiven-kafka-connect-jdbc` plugin to version `6.9.0` affects both the JDBC source
+and sink connectors.
+
+:::note
+Pinning a plugin version restarts Apache Kafka Connect and reloads all plugins.
+Allow time for the process to complete.
+:::
+
+<Tabs groupId="setup-method">
```

Comment:
Let's use the same id as above so that the dev tool tabs sync on ppl's initial selection 🙂 

```suggestion
<Tabs groupId="check-method">
```